pwiz/analysis/spectrum_processing/SpectrumList_IonMobility.cpp

```diff
@@ -79,13 +80,13 @@ SpectrumList_IonMobility::SpectrumList_IonMobility(const msdata::SpectrumListPtr
         equipment_ = IonMobilityEquipment::UIMFDrift;
         units_ = IonMobilityUnits::drift_time_msec;
     }
-    else // reading an mzML conversion?
+    else // reading an mzML conversion
     {
         if (inner->size() == 0)
             return;
 
         // See if first scan has any ion mobility data
-        SpectrumPtr spectrum = inner->spectrum(0, false);

```

Comment:
Isn't it pretty common for the first Waters scan to be lockmass? I could easily see that being non-IMS.

Reply:
Might be better to check the first several, yes.